nhais/common/configuration/configuration_manager.py

```diff
@@ -0,0 +1,23 @@
+"""This module defines the configuration manager component."""
+import copy
+
+from utilities.file_utilities import FileUtilities
+
+
+class ConfigurationManager(object):
```

Comment:
NAHIS won't be loading config from a JSON file like this. I don't think its needed.